data/escher-p2.mesh

```diff
@@ -115,7 +115,7 @@ vertices
 
 nodes
 FiniteElementSpace
-FiniteElementCollection: Quadratic
+FiniteElementCollection: H1_3D_P2
```

Comment:
Maybe now is the time to finally do this, but lets bring it up at the PR meeting